cfg_mgmt/rotate.py

```diff
@@ -11,6 +18,89 @@
 logger = logging.getLogger(__name__)
 
 
+def write_config_queue(
+    cfg_queue: typing.Iterable[cmm.CfgQueueEntry],
+    cfg_queue_file_path: str,
+):
+    with open(cfg_queue_file_path, 'w') as f:
+        yaml.dump(
+            {
+                'rotation_queue': [
+                    dataclasses.asdict(cfg_queue_entry)
+                    for cfg_queue_entry in cfg_queue
+                ]
+            },
+            f,
+        )
+
+
+def push_config_queue(
```

Comment:
rather inline the function. also, the name is not a good fit (apart from commit-msg, this function does not focus on interacting w/ config-queue